src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import helper.*;
+
+import helper.command.Command;
+
+import java.util.Scanner;
```

Comment:
Maybe you want to try checkStyle? It will adjust the import order to the correct order. But anyway, this course does not specify the import order.